### ISSUE TITLE
Authentication errors are not caught early.

### ISSUE THREAD

**Issue by symbiont-aurelien-bonnel** [symbiont-io/symenv#8]
```shell
symenv remote

parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10
```

Force authenticating or deleting the `.symenvrc` file reset the state correctly.

**Comment by symbiont-jacob-romer** [comment]
Also: Return unmistakeable warning if user hasn't set registry and default registry doesn't work correctly.

**Comment by symbiont-jacob-romer** [comment]
To be addressed in #13

**Comment by symbiont-jacob-romer** [comment]
resolved in #13
